src/test_linked_list.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding utf-8 -*-
+import pytest
+
+
+def test_push():
+    from linked_list import LinkedList
+    ll = LinkedList()
+    ll.push('bob')
+    assert ll.head.get_data() == 'bob'
+
+
+def test_pop():
+    from linked_list import LinkedList
+    ll = LinkedList()
+    ll.push('bob')
+    assert ll.pop() == 'bob'
+
+
+def test_pop_empty():
+    from linked_list import LinkedList
+    ll = LinkedList()
+    with pytest.raises(IndexError):
+        ll.pop()
+
+
+def test_size():
+    from linked_list import LinkedList
+    ll = LinkedList()
+    ll.push('bob')
+    ll.push('fred')
+    assert ll.size() == 2
```

Comment:
What about if you push only one thing? Is the size 1? What if you make a new LinkedList? What's the size of that? What if you push 10 things? What's the size of that list?